src/Router.class.js

```diff
@@ -1,15 +1,20 @@
+// Import Internal Dependencies
+import logger from "../logger.js";
+
 export default class Router {
   routes = new Map();
   stop = false;
 
   register(name, handler) {
     if (typeof handler !== "function") {
+      logger.error("Handler must be a function");
```

Comment:
Uselss log, it doesn't give more informations compared to the error thrawn.
You could improve with something like:
```suggestion
      logger.error(`[Router:register](route: ${name}, expected handler: function, given: ${typeof handler})`);
```